fleet/demo/stress-e2e.py

```
@@ -0,0 +1,367 @@
+                          getattr(sandbox, "name", None) or \
+                          getattr(sandbox, "sandbox_name", None)
+    except SandboxNotReadyError as e:
+      result.error = f"not ready within {timeout_s}s: {e}"[:200]
+    except Exception as e:  # noqa: BLE001 — surface everything else
+      result.error = f"{type(e).__name__}: {e}"[:200]
+    finally:
+      if cleanup and result.claim_name:
+        try:
+          fleet.delete_sandbox(result.claim_name, namespace=NAMESPACE)
+        except Exception:  # noqa: BLE001
+          pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)